src/cli/input-from-terminal.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const readline = require("readline");
+const { stdin: input } = require("node:process");
+const { printToScreen } = require("./utils.js");
+
+/**
+ * This function starts reading any input which user types on the console.
+ * After the user clicks ctrl+D to exit, the input is formatted and printed
+ * on the screen.
+ * @returns {Promise<string>} On success, resolves the input code joined together
+ * to form a single string. On error, reject with an error message
+ */
+function getInputFromTerminal() {
+  return new Promise((resolve, reject) => {
+    printToScreen("Start writing your code snippet below");
+    printToScreen("After finishing press ctrl+D to exit the read mode");
+    const inputCode = [];
+    const readLine = readline.createInterface({ input });
+    readLine.on("line", (line) => inputCode.push(line));
+    readLine.on("close", () => {
+      if (inputCode.length > 0) {
+        printToScreen("Formatted code");
```

Comment:
This should print after the code if formatted.

Reply:
so you mean something like this
```console
prettier --stdin --parser babel
Start writing your code snippet below
After finishing press ctrl+D to exit the read mode
hello

hello;
Formatted code
```

Reply:
Pushed changes with the above pattern.

Reply:
Sorry, I mean formatted, but  before print.

Reply:
I am sorry, I still do not get it. Can you drop a suggestion please?

Reply:
https://github.com/prettier/prettier/pull/13428/commits/d856d63a9acfefc12586d402589b8af3cfbda3c6